internal/api/v1/user/accesskeys/accesskeystest/access_keys_api_fixtures.go

```diff
@@ -85,3 +88,31 @@ func ExpectedAccessKeyPage(t *testing.T) *queries.AccessKeyPage {
 		},
 	}
 }
+
+func Ptr[T any](value T) *T {
```

Comment:
Most of the functions have been moved into the spvwallettest package. To avoid import loop cycles, some of them—particularly those related to non-exported package artifacts like structures—may need to be duplicated. This approach ensures the code remains decoupled and avoids circular dependencies.

Reply:
also there is package `ptr.To` `k8s.io/utils/ptr`